### PR TITLE
Modify ibft setup to showcase adding the bootnodes

### PR DESCRIPTION
**Pull request by Vuksan**
PR that adds this functionality to Polygon SDK - [link](https://github.com/0xPolygon/polygon-sdk/pull/47).

Ignore the `build` directory, all essential changes are here: `docs/how-tos/howto-set-ibft.md`

_Note:_ I didn't update `howto-set-up-node` since it looked outdated (it mentions `pow`...), and isn't listed in the sidebars. If you think that I should do that anyway, please let me know.

### CODE REVIEW

**Review by zivkovicmilos — APPROVED** [review]
Looks good to me 👍 
There may be some unused files - we can safely remove them, as you mentioned.
If it's not in the sidebars.js file, it can be removed 💯 %

File: docs/how-tos/howto-set-ibft.md [hunk 1]
```diff
@@ -39,10 +39,27 @@ go run main.go ibft init test-chain-3
...
+
+```
+/ipv4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW
+```
+
+## Step 3: Generate an IBFT genesis file with the previous accounts as validators
 
 ````bash
-go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain-
+go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ipv4/127.0.0.1/tcp/10001/p2p/<node_id_1> --bootnode /ipv4/127.0.0.1/tcp/20001/p2p/<node_id_2> --bootnode /ipv4/127.0.0.1/tcp/30001/p2p/<node_id_3> --bootnode /ipv4/127.0.0.1/tcp/40001/p2p/<node_id_4>
```
Comment (Kourin1996) [inline docs/how-tos/howto-set-ibft.md:62]: `ip4` is correct protocol name in libp2p

```suggestion
go run main.go genesis --consensus ibft --ibft-validators-prefix-path test-chain- --bootnode /ip4/127.0.0.1/tcp/10001/p2p/<node_id_1> --bootnode /ip4/127.0.0.1/tcp/20001/p2p/<node_id_2> --bootnode /ip4/127.0.0.1/tcp/30001/p2p/<node_id_3> --bootnode /ip4/127.0.0.1/tcp/40001/p2p/<node_id_4>
```

**Review by Kourin1996 — CHANGES_REQUESTED** [review]

File: docs/how-tos/howto-set-ibft.md (hunk 1 above)
Comment (Vuksan) [reply docs/how-tos/howto-set-ibft.md:62]: Omg, GJ for noticing that 🍻

**Review by Kourin1996 — APPROVED** [review]
LGTM. Thank you for the fix!

File: docs/how-tos/howto-set-ibft.md [hunk 2]
```diff
@@ -39,10 +39,27 @@ go run main.go ibft init test-chain-3
...
+into a [multiaddr format](https://docs.libp2p.io/concepts/addressing/):
+```
+/ipv4/<ip_address>/tcp/<port>/p2p/<node_id>
+```
+
+To do so, you should specify the IP address and the port for each of the clients,
+and append the node ID (from the previous step) to it. For example:
+
+```
+/ipv4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW
```
Comment (lazartravica) [inline docs/how-tos/howto-set-ibft.md:56]: `/ip4/127.0.0.1/tcp/10001/p2p/16Uiu2HAmJxxH1tScDX2rLGSU9exnuvZKNM9SoK3v315azp68DLPW`

Comment (lazartravica) [reply docs/how-tos/howto-set-ibft.md:56]: Replace ipv4 with ip4

**Review by lazartravica — APPROVED** [review]
LGTM! Just replace the `ipv4` with `ip4` and that's it :)

File: docs/how-tos/howto-set-ibft.md (hunk 2 above)
Comment (Vuksan) [reply docs/how-tos/howto-set-ibft.md:56]: 🤦

**Pull request merged**